app/app/users/[user_id]/page.tsx

```diff
@@ -51,17 +57,52 @@ export default function UserDetails({
 }) {
   const token = useToken();
   const userId = params.user_id;
-  const { data: user } = useUser(userId, token);
+  const { data: user, mutate } = useUser(userId, token);
+  const [showNewUserModal, setShowNewUserModal] = useState(false);
+  const onCloseModal = () => setShowNewUserModal(false);
+
+  const onUpdateUsercallback = useCallback(async () => {
```

Comment:
should the capitalization here be -> `onUpdateUserCallback`

Reply:
yep—thanks!